include/dlpack/dlpack.h

```diff
@@ -26,12 +29,6 @@
 #define DLPACK_DLL
 #endif
 
-#ifdef __cplusplus
-#define DLPACK_INT32_ENUM_ANNOTATION : int32_t
-#else
-#define DLPACK_INT32_ENUM_ANNOTATION
-#endif
```

Comment:
I had to remove this because Doxygen doesn't parse the expression `typedef enum DLPACK_INT32_ENUM_ANNOTATION  { ...` correctly. It thinks that the enum is named `DLPACK_INT32_ENUM_ANNOTATION` which is wrong and generates erroneous XML index file. To address this, I just moved the annotation inline below (line 41-45) which gets parsed correctly by Doxygen. I thought it's OK to do this since `DLPACK_INT32_ENUM_ANNOTATION` wasn't used anywhere else. Let me know if this is OK.

Reply:
That sounds reasonable change to me